### DISCUSSION TITLE
Create single JS file without require for browser.

### DISCUSSION THREAD

**Discussion by DiachenkoIgor** [parcel-bundler/parcel#6309] [Q&A] [answered]
Hi, I am new to Parcel and I can't create single js file with all dependencies in one file. I use such command for building:
`parcel build --target browser main.ts --bundle-node-modules`
Result file contains "requires" for libraries. Please, help.

**Comment by mischnic** [comment, marked as answer]
This should work (and `--target browser --bundle-node-modules` are the default options anyway, so it's the same as `parcel build main.ts`)

**Reply by DiachenkoIgor** [reply]
However, when I run js file (without server, because It has to be single) I receive error:

> Uncaught ReferenceError: regeneratorRuntime is not defined
    at main.js:2
    at main.js:2
    at Object.parcelRequire.QaZ1 (main.js:2)

**Reply by mischnic** [reply]
One possible solution for that: https://github.com/parcel-bundler/parcel/issues/1762#issuecomment-504389468
